osu.Game/Screens/Play/PlayerInputManager.cs

```diff
@@ -48,7 +48,9 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             parentClock = Clock;
+            clock.CurrentTime = parentClock.CurrentTime;
             Clock = new FramedClock(clock);
+            Clock.ProcessFrame();

```

Comment:
this should not be necessary. if a framed clock is initialised with an underlying clock, it should receive the clock's value without a processframe call. this is a hack.